base/client.jl

```diff
@@ -433,6 +433,17 @@ function run_main_repl(interactive::Bool, quiet::Bool, banner::Bool, history_fil
     nothing
 end
 
+"""
+    activate_module(mod::Module=Main)
+
+Set `mod` as the default contextual module in the REPL,
+both for evaluating expressions and printing them.
+"""
+function activate_module(mod::Module=Main)
```

Comment:
Maybe we can make this a bit shorter? Just for convenience if you want to swap module often. Could be just `activate` since the module is implicit with the argument type (would also mirror `Pkg.activate`?

Reply:
`activate` sounds fine but then it might be preferable to not export it (i.e. call it `Base.activate`) as `activate` is quite a generic verb?

Reply:
Oh, I was sure this was in the `REPL` stdlib but it is not. I really think it should be `REPL.activate`.

Reply:
Yeah, I also thought `REPL.activate` would be nice after reading your comment, but it updates `Base.active_repl` ... But sure it's easy enough to call it `REPL.activate` :)

Reply:
Yeah, but that is arguably irrelevant from a users point of view.